packages/whats-new/src/index.js

```diff
@@ -1,18 +1,11 @@
-/**
- * External dependencies
- */
+import { useLocale } from '@automattic/i18n-utils';
 import { Guide } from '@wordpress/components';
-import proxyRequest from 'wpcom-proxy-request';
-import React, { useEffect, useState } from 'react';
 import { useI18n } from '@wordpress/react-i18n';
-import { useLocale } from '@automattic/i18n-utils';
+import React, { useEffect, useState } from 'react';
 import wpcom from 'wpcom';
-
-/**
- * Internal dependencies
- */
-import './style.scss';
+import proxyRequest from 'wpcom-proxy-request';
 import WhatsNewPage from './whats-new-page';
+import './style.scss';
```

Comment:
Not sure if it's relevant @scinos but the linter doesn't complain if I add an empty line over the style import (in fact it was there after running the `import-order` script).

Reply:
My rule of thumb is that eslint+prettier are fine with the empty lines (or the lack thereof), so am I.

I've been moving the unassigned imports together after the import block (when it is obvious it won't break anything, like a CSS) and separating them with an empty line from the import block, but that's a personal preference.

The doc for `import/order` says "Unassigned imports are ignored, as the order they are imported in may be important.". But I think it will assume the "import block" ends with the first unassigned import, so empty lines between the "import block" and the unassigned import are fine.

It definitely a bit buggy when it finds an unassigned import in the middle of the import block. I had a couple of PRs where I had to run the script twice so imports where correctly sorted after the unassigned import.